tests/e2e/taqueria-plugin-metadata.spec.ts

```diff
@@ -1,208 +1,155 @@
-import { exec as execRaw, spawn } from 'child_process';
-import fsPromises from 'fs/promises';
+import { exec as exec1 } from 'child_process';
 import util from 'util';
-import { generateTestProject } from './utils/utils';
-const exec = util.promisify(execRaw);
-
-const taqueriaProjectPath = 'scrap/auto-test-metadata-plugin';
+const exec = util.promisify(exec1);
+import path from 'path';
+
+import { prepareEnvironment } from '@gmrchk/cli-testing-library';
+
+describe('Metadata Plugin E2E Testing for the Taqueria CLI', () => {
+	jest.setTimeout(120000);
+
+	// these tests are broken on dev-main as the metadata request does not realize there is an imported plugin for it
+
+	test.skip('generate-project-metadata will add a metadata entry to the config.json', async () => {
+		const { execute, spawn, cleanup, ls, writeFile, readFile } = await prepareEnvironment();
+		const { waitForText } = await spawn('taq', 'init test-project');
```

Comment:
Btw do you mean to install the "core" plugin from NPM or local? If local, then provide --debug flag. If not this is fine.

This applies to all taq inits in e2e tests.

Note: "core" plugin is installed automatically upon taq init. That's why I wanted to confirm.

Reply:
Thanks Jimmy !!